src/components/Event/docs/EventWiringExample.st.css

```
@@ -0,0 +1,19 @@
+:import {
+    -st-from: "../Event.st.css";
+    -st-default: TPAEvent;
+}
+
+.root {
+    -st-mixin: TPAEvent(
+            TimeColor '"--timeColor"',
+            TitleColor '"--titleColor"',
+            AllDayBackground '"--allDayBackground"',
+            FontStyle '"--fontStyle"',
+            AllDayOpacity '"unit(fallback(zeroAsTrue(--allDayOpacity), 0.2), "")"'
+    );
+}
+
+.flexContainer {
+    display: flex;
+    justify-content: space-around;
+}
```

In accordance with what I wrote in `EventWiringExample.tsx`
The correct way to write this would be:
```suggestion
.root {
    display: flex;
    justify-content: space-around;
}

.event {
    -st-mixin: TPAEvent(
            TimeColor '"--timeColor"',
            TitleColor '"--titleColor"',
            AllDayBackground '"--allDayBackground"',
            FontStyle '"--fontStyle"',
            AllDayOpacity '"unit(fallback(zeroAsTrue(--allDayOpacity), 0.2), "")"'
    );
}
```